### ISSUE TITLE
Missing archive standards documents, labels, and DOIs for 1.22.0.0 and 1.23.0.0

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/portal-tasks#112]
### 💡 Description

All standards docs are missing from https://pds.nasa.gov/data/pds4/documents/document_pds4_standards/1.20.0.0/

- [x] Generate appropriate labels and DOIs (@rsjoyner )
- [x] Post online in archive data area (@c-suh)
- [ ] Update wiki procedures to include this step (@c-suh and @jordanpadams)
- [x] Update website to point to these versions of the documents, not some other versions on the website.

**Comment by rsjoyner** [comment]
Enclosed are the 2 zip files for the Standard Docs Collection for both 1.22.0 and 1.23.0
Let me know if you need more info.

[Build_22_20240331.zip](https://github.com/user-attachments/files/18398654/Build_22_20240331.zip)
[Build_23_20241204.zip](https://github.com/user-attachments/files/18398655/Build_23_20241204.zip)

**Comment by c-suh** [comment]
Hi @rsjoyner - thanks for the files! I will take a look at these and let you know if I've any questions.

**Comment by jordanpadams** [comment]
@rsjoyner it looks like the labels for Build_22 have a few bugs:

[validate.log.txt](https://github.com/user-attachments/files/18510097/validate.log.txt)

And the Build_23 ZIP only contains a collection label?

**Comment by c-suh** [comment]
@jordanpadams yup, I've been emailing with Ron about the Build 23 zip. Is there something that I have to (re)do for the Build 22 files?

**Comment by c-suh** [comment]
Hi @jordanpadams, for the 4th checklist item, "Update website to point to these versions of the documents, not some other versions on the website.", do you mean, for example, that all links on the [DPH page](https://pds.nasa.gov/datastandards/documents/dph/) should point to the DPH pdfs at https://pds.nasa.gov/data/pds4/documents/document_pds4_standards/?

If so, should this be for **all** the versions or only starting from a certain point (e.g., version 1.15.0.0, which has a directory in the archive data area)?

If for all, I will create the missing directories (e.g., 1.0.0.0, 1.4.1.0) and move the corresponding files. Do I ask Ron if he has or can create labels for those files?

**Comment by rsjoyner** [comment]
> [@rsjoyner](https://github.com/rsjoyner) it looks like the labels for Build_22 have a few bugs:
> 
> [validate.log.txt](https://github.com/user-attachments/files/18510097/validate.log.txt)
> 
> And the Build_23 ZIP only contains a collection label?

@jordanpadams @c-suh @rchenatjpl 

(1) The "errrrrrrors" from Validate in the zip are because I didn't include the PDFs in the zip.   I send the docx and PDF in a separate email.   I can combine them if that help this process.   See enclosure for the PDFs for the Build-22 and Build-23 docs.   My preference is that I only update the XML labels for the PDF version of the 3 docs.
(2) The zip contains all of the XML labels for each of the 3 docs and the versions of the docs ????

After a lot of back-and-forth, I think this is closed ???

If not, let me know the next steps....

**Comment by jordanpadams** [comment]
@c-suh don't worry about older versions of the DPH/standards reference right now. Eventually we will want to go back and generate all of that... but not required right now. Maybe we can create a ticket to pursue that down the road.

**Comment by c-suh** [comment]
Need clarification from Jordan about the process. Will do this via Slack.

**Comment by jordanpadams** [comment]
@c-suh 

1. What is the process for posting these documents?

For future builds (not this ticket), documents can be posted on release day. If you get them early, you can hold them until release day.

2. Where should the links go?

Let's have the links go to the archive from now on. Avoids the extra overhead of maintaining the symlinks.


For this ticket, you can post the documents Ron gave you as soon as you are ready because they are already released.

**Comment by tloubrieu-jpl** [comment]
@c-suh is updating the documentation.

**Comment by jordanpadams** [comment]
@c-suh this is done, no?

**Comment by c-suh** [comment]
@jordanpadams not technically, no. Sorry for the delay. I need to review the documentation again, but also the very last checklist item isn't done. For example, the .pdfs for versions 1.15.0.0 and later on [this DPH page](https://pds.nasa.gov/datastandards/documents/dph/) should be linked to the respective files in the [archive directory](https://pds.nasa.gov/data/pds4/documents/document_pds4_standards/). I will wrap this up after I&T testing.

**Comment by c-suh** [comment]
I've updated the links to use the archived files, but am going to review the documentation one more time before closing this issue.

**Comment by c-suh** [comment]
Documentation updated! Any holes will be revealed in the next build. This task is done for now.
